[2024-06-27T15:04:34.712+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T15:04:34.928+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_sample_dag.task_from_script2 scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2024-06-27T15:04:35.053+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_sample_dag.task_from_script2 scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2024-06-27T15:04:35.077+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-27T15:04:35.267+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_from_script2> on 2023-01-03 00:00:00+00:00
[2024-06-27T15:04:35.320+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_sample_dag', 'task_from_script2', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpy4dxritm']
[2024-06-27T15:04:35.317+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=263) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-27T15:04:35.329+0000] {standard_task_runner.py:63} INFO - Started process 491 to run task
[2024-06-27T15:04:35.328+0000] {standard_task_runner.py:91} INFO - Job 38: Subtask task_from_script2
[2024-06-27T15:04:35.873+0000] {task_command.py:426} INFO - Running <TaskInstance: my_sample_dag.task_from_script2 scheduled__2023-01-03T00:00:00+00:00 [running]> on host dccfc2b642a8
[2024-06-27T15:04:36.755+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_sample_dag' AIRFLOW_CTX_TASK_ID='task_from_script2' AIRFLOW_CTX_EXECUTION_DATE='2023-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-03T00:00:00+00:00'
[2024-06-27T15:04:36.762+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T15:04:38.203+0000] {logging_mixin.py:188} INFO - Accuracy: 0.9957285239677266
[2024-06-27T15:04:38.214+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-27T15:04:38.219+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T15:04:38.276+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_sample_dag, task_id=task_from_script2, run_id=scheduled__2023-01-03T00:00:00+00:00, execution_date=20230103T000000, start_date=20240627T150434, end_date=20240627T150438
[2024-06-27T15:04:38.973+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-27T15:04:39.090+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T15:04:39.103+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
